bindings/python/System/src/Module.cpp

```diff
@@ -9,6 +9,7 @@
 
 #include <BipedalLocomotion/bindings/System/Module.h>
 #include <BipedalLocomotion/bindings/System/VariablesHandler.h>
+#include <BipedalLocomotion/bindings/System/LinearTask.h>
```

Comment:
Nit: it seems you are not using clang-format even though I did not notice evident style problem in this PR. For example, I suspect that headers are alphabetically ordered, and this is not. Can you double check if your IDE detects the clang-format file?

Reply:
I am using pycharm for the development of the bindings and, after a quick search, I didn't find how to use clang-format in pycharm. If you have suggestions, they are welcome.

Reply:
Ow no, you have the awesome qtcreator in the docker image you're using. Just open the CMakeLists.txt and check if the existing build folder is detected. Then, the clang-format file should be automatically detected (or, check the qtcreator plugins in the options to enable the support).

Reply:
Thanks @diegoferigo, qtcreator worked. 

However, when I open the blf CMakeLists.txt - in my current configuration - the resulting project does not include the files in the bindings folder (apart from CMakeLists.txt files). Therefore, I need to open the files to reformat one by one. Sorry in advance for my poor knowledge of qtcreator.

Reply:
No worries. If qtcreator detected the correct build folder, the project configured in that build folder must have the python bindings enabled. Qtcreator will detect the new CMake targets with the corresponding files, and show them in the panel.

<details>
<summary>Image</summary>

![Screenshot_20210518_123158](https://user-images.githubusercontent.com/469199/118636683-3b8dd500-b7d5-11eb-8e5e-07a650d98b24.png)

</details>

Reply:
Addressed in https://github.com/dic-iit/bipedal-locomotion-framework/pull/303/commits/ab5bb3da2ec77f259378a22935ca06791f01813a